.github/workflows/ci.yml

```diff
@@ -553,3 +553,18 @@ jobs:
 
       - name: Output Coveralls response
         run: echo ${{ steps.publish-coverage.outputs.coveralls-api-result }}
+
+  slack-on-fail:
+    needs:
+      - cleanup-matrix
+    runs-on: ubuntu-latest
+    steps:
+       - name: Slack on fail
+         id: send-slack-message
+         if: false #failure() && ${{ github.head_ref == "" }}
+         uses: act10ns/slack@v2.0.0
+         with:
+              webhook-url: ${{ secrets.SLACK_WEBHOOK_URL }}
```

Comment:
@nirinchev can you help us with setting `secrets.SLACK_WEBHOOK_URL` to github secrets?